holoviews/plotting/bokeh/raster.py

```diff
@@ -20,9 +20,12 @@
 from .util import BOKEH_GE_3_3_0, BOKEH_GE_3_4_0, colormesh
 
 
-class ServerHoverMixin:
+class ServerHoverMixin(param.Parameterized):
     _model_cache = {}
 
+    selector_in_hovertool = param.Boolean(default=True, doc="""
```

Comment:
I'm not a big fan of this name, but I'm open to suggestions.

Reply:
I think "show_selector" is a bit better